content/guides/forecasts.md

```diff
@@ -0,0 +1,92 @@
+
+Navigate to the [New Monitor page](https://app.datadoghq.com/monitors#create/forecast) for **Forecast Alerts**. Then fill out the **Define the metric** section just as you would for any other metric monitor.
+
+{{< img src="forecasts/alert_conditions.png" >}}
+There are three required options for setting up a forecast alert:
+
+<ol type="a">
+  <li> This is the threshold at which an alert is triggered. For a metric like `system.disk.in_use` this should be set to 1.0, whereas for a metric like `system.mem.pct_usable` this should be set to 0.0. A recovery threshold is also required.
+  <li> This is the the condition on which an alert is triggered. For a metric like `system.disk.in_use` this should be set to “above or equal to”, whereas for a metric like `system.mem.pct_usable` this should be set to “below or equal to”.
+  <li> Control how far in advance you would like to be alerted before your metric hits its critical threshold.
+</ol>
+
+{{< img src="forecasts/alert_advanced.png" >}}
+
+Datadog will automatically set the **Advanced** options for you by analyzing your metric. Note that any changes in the **Define the metric** section could change the advanced options.
+
+<ol type="a">
+  <li> You can change the forecasting algorithm to be used here. See the next section of this guide for tips on how to choose the best algorithm for your use case. Each algorithm also has additional settings that will be described in the next section.
+  <li> We recommend using larger intervals between points to avoid having noise influence the forecast too much.
+  <li> The number of deviations controls the width of the range of forecasted values. A value of 1 or 2 should be large enough to accurately forecast most “normal” points.
+
+Complete all steps in the New Monitor form (**Say what’s happening**, etc.) and click **Save** to create the Forecast monitor.
+
+Both the Monitor Edit page and the Monitor Status pages provide “Historical Context” so that you can see how the metric behaved in the past. This should provide some insight into what the forecast algorithm takes into account when predicting future values.
+
+### Forecast Algorithms
+
+There are two different forecast algorithms:
+
+**Linear**: Use this algorithm for metrics that have no repeating seasonal pattern, and tend to have steady trends. On dashboards, linear will use the data within view to create a forecast of the same length. E.g., if the time selector is set to “The Past Week”, the forecast will use the past week of data to forecast the next week. For monitors, you can explicitly set the amount of history that is used, and it is set to one week by default.
+
+{{< img src="forecasts/linear.png" >}}
+
+The linear algorithm has three different _models_ that control how sensitive the algorithm is to level shifts.
+
+The “simple” model does a robust linear regression through the entire history.
```

Comment:
I don't know if you answered my comment in the Google Doc, but I think we should put default first because it is the one that is used by default and because simple and reactive should serve more specific purposes. I imagine someone who looks at this section might do so because they are not entirely happy with their current forecast and they want to adjust the algorithm, so I think it's better to introduce default first ("this is the algo you've been using so far"), and simple and reactive afterwards ("those are models that can fit your more specific needs").
But that's not a big deal anyway, if you prefer the current way.

Reply:
oh sorry, forgot to respond. I couldn't think of a good way to frame it without introducing simple and reactive first.